### ISSUE TITLE
MC 1.20.1 Fabric, shield don't block and skills don't work.

### ISSUE THREAD

**Issue by Kavaltheone** [nvb-uy/Shield_Overhaul#24]
Hello.
I've seen similar issues, but I think they are not fully describe the bug. So I made a video to show how vanilla shield works (or doesn't work) and compared to how work shields from mod Basic Shields. It's pretty short and I hope it will help to see the problem. In short, iron shield don't block when main hand is not empty, skills don't do damage and don't apply effect. And I think shield item removes from player and then adds to player on RMB and it's hardly a good idea.

https://youtu.be/hd-zqf65YOg

**Comment by DrThraxGLA** [comment]
I agree as well with forge also it doesn't show up in game mod list so I can't block keep getting hit even with holding up shield it doesn't block it and skill don't work there not even keybind.

**Comment by KingYamiJ** [comment]
The shield only works when there are no weapons in the main hand for me. (1.20.1)

**Comment by nvb-uy** [comment]
Disable parrying, Joe is working on the update but he’s been very busy
